website/blog/2024-09-04-launch-week-2/index.mdx

```diff
@@ -0,0 +1,112 @@
+
+Yeehaw!
+
+> Ok I’ll stop talking like a cowboy now 😅
+
+I hope it goes without saying that all are welcome, so don’t be a stranger, join us on Monday the 9th of September for our launch week kickoff party. **Reserve your spot** [here](https://discord.gg/Jswn2Vqz?event=1274009068884721725).
+
+![Discord Launch Party](https://github.com/user-attachments/assets/b9773cf0-03bf-4fa4-9dbd-b2bf2a887e38)
+
+Make sure to mark “**interested**“ to be notified on the day of the event.
+
+So you might be thinking to yourself, what does the Glasskube team have up their sleeve? Let’s have a look, shall we?
+
+## 1\. Glasskube is Gitops Ready with ArgoCD
+
+Day one of launch week will be about celebrating the fact that it’s safe to consider Glasskube “[GitOps ready](https://glasskube.dev/docs/integration/gitops/)“. This might not seem like such a big deal to folks not experienced in DevOps and Cloud engineering practices but experienced Kubernetes administrators will quickly pick up on how much of a big deal this is.
+
+![GitOps ready](https://github.com/user-attachments/assets/491b146a-63a0-4585-93ee-aac8b7013f53)
+
+During the rest of the launch week, we will showcase a Glasskube feature that adds key components to the GitOps completeness of Glasskube.
+
+## 2\. What’s so good about GitOps anyway?
+
+Imagine if I told you you could set up a full Kubernetes setup with just one command. Glasskube bootstrap command
+
+![Glasskube GitOps command](https://github.com/user-attachments/assets/3dab678a-1ff7-4167-82b6-975070c70a0d)
+
+The Glasskube GitOps template is [officially here](https://glasskube.dev/blog/gitops-template/), think of it as a package for your packages. You can now define every single component of your cluster package stack, push it to a Git repository, and let ArgoCD handle the deployment.
+
+And it’s no exaggeration to say that deploying to Kubernetes now takes seconds instead of hours. Who wouldn’t want that?
+
+![GitOps template diagram](https://github.com/user-attachments/assets/236af5f2-64a9-42f2-92f8-a30438777365)
+
+## 3\. Hear that? …silence! But your packages are still up-to-date
+
+You might be wondering, "Wait, so I don’t have to update the packages myself?"
+
+That's right! The Glasskube package repository, the GitHub repo that stores all the packages on Glasskube Hub, runs a [Renovate](https://docs.renovatebot.com/) integration on the backend repo every day. If new versions of supported Glasskube packages are available, we test and add them to the list of versions you can choose from.
+
+![Renovate integration](https://github.com/user-attachments/assets/afc36885-6781-4cda-b8e7-7e1de59902ef)
+
+Well dive deeper into [how this works](https://glasskube.dev/docs/integration/renovate/) and discuss how you can opt in or out of the auto-updating feature.
```

Comment:
we could also link the upstream url: https://glasskube.dev/docs/integration/renovate/